 # A camera to fiducial transform with supporting data

 Header header

 int32 image_seq
 int32 fiducial_id

 geometry_msgs/Transform transform
 float64 image_error
 float64 object_error